engineering/licensing.md

```diff
@@ -0,0 +1,37 @@
+# Licensing Policy
+
+This document describes the source{d} licensing policy for the open-source
```

Comment:
The lines are semantic I am not splitting the lines based on how they looks are full phrases.